crypto/types/multisig/threshold_pubkey_test.go

```diff
@@ -135,8 +135,8 @@ func TestThresholdMultisigDuplicateSignatures(t *testing.T) {
 }
 
 func TestMultiSigPubKeyEquality(t *testing.T) {
-	pubKey1 := secp256k1.GenPrivKey().PubKey()
-	pubKey2 := secp256k1.GenPrivKey().PubKey()
+	pubKey1 := &keys.Secp256K1PubKey{Key: secp256k1.GenPrivKey().PubKey().(secp256k1.PubKey)}
+	pubKey2 := &keys.Secp256K1PubKey{Key: secp256k1.GenPrivKey().PubKey().(secp256k1.PubKey)}
```

Comment:
this makes the test fail now because regular amino pub keys should be used in the other test cases.

Reply:
Ah correct. Fixed in 99a15558e23b7a030a757d37212d741f195f73b2 (note that lines 173-4 have also been changed).